### PR TITLE
Cortex XDR - possible external RDP brute force

### PR DESCRIPTION
**Pull request by ArikDay**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-4678)

## Description
Possible External RDP Brute Force - Playbooks, Layout, Fields.

## Screenshots
Cortex_XDR_-Possible_External_RDP_Brute_Force_-_Set_Verdict
Cortex_XDR_Alerts_Handling
TIM_-_Indicator_Relationships_Analysis
Cortex_XDR_-_Possible_External_RDP_Brute-Force

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.8.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@altmannyarden @melamedbn Doc review completed.

File: Packs/CommonTypes/IncidentFields/incidentfield-Endpoint_Isolation.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "endpointisolation",
```
Comment (melamedbn) [inline Packs/CommonTypes/IncidentFields/incidentfield-Endpoint_Isolation.json:4]: Change the name to a name that can easily hint at the field purpose.

File: Packs/CommonTypes/IncidentFields/incidentfield-IP_Blocked_Status.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "ipblockedstatus",
```
Comment (melamedbn) [inline Packs/CommonTypes/IncidentFields/incidentfield-IP_Blocked_Status.json:4]: Maybe IP Address Status? with the value of Blocked/Not Blocked?

File: Packs/CommonTypes/IncidentFields/incidentfield-IP_Reputation.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "ipreputation",
```
Comment (melamedbn) [inline Packs/CommonTypes/IncidentFields/incidentfield-IP_Reputation.json:4]: Why do we need an IP Reputation field?

File: Packs/CommonTypes/IncidentFields/incidentfield-Password_Changed.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "passwordchanged",
```
Comment (melamedbn) [inline Packs/CommonTypes/IncidentFields/incidentfield-Password_Changed.json:4]: Change the name to a name that can easily hint at the field purpose.

File: Packs/CommonTypes/IncidentFields/incidentfield-User_Engagement.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "userengagement",
```
Comment (melamedbn) [inline Packs/CommonTypes/IncidentFields/incidentfield-User_Engagement.json:4]: Change the name to a name that can easily hint at the field purpose.

File: Packs/CortexXDR/IncidentFields/incidentfield-XDR_Alert_Search_Results.json [hunk 6]
```diff
@@ -0,0 +1,164 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "xdralertsearchresults",
```
Comment (melamedbn) [inline Packs/CortexXDR/IncidentFields/incidentfield-XDR_Alert_Search_Results.json:4]: Wouldn't XDR Search Alerts Results be more accurate?

Comment (ArikDay) [reply Packs/CortexXDR/IncidentFields/incidentfield-XDR_Alert_Search_Results.json:4]: "XDR Search Alerts Results" implies that i'm searching for results that related to alert. what i'm doing is searching for alerts.

File: Packs/CommonTypes/IncidentFields/incidentfield-IP_Reputation.json (hunk 3 above)
Comment (ArikDay) [reply Packs/CommonTypes/IncidentFields/incidentfield-IP_Reputation.json:4]: showing the IP reputation in the layout.

File: Packs/CommonTypes/IncidentFields/incidentfield-IP_Blocked_Status.json (hunk 2 above)
Comment (ArikDay) [reply Packs/CommonTypes/IncidentFields/incidentfield-IP_Blocked_Status.json:4]: Given that we cannot know for sure that the IP was blocked, the value will be "Executed". maybe "IP Block" will be better

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml [hunk 7]
```diff
@@ -0,0 +1,3115 @@
+id: Cortex XDR - Possible External RDP Brute-Force
```
Comment (melamedbn) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml:1]: Go over all tasks/playbooks that aren't CommonPlaybooks/Scripts and mark the check box of 'skipunavailable'
image

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force_-_Set_Verdict.yml [hunk 8]
```diff
@@ -0,0 +1,557 @@
+id: Cortex XDR - Possible External RDP Brute-Force - Set Verdict
```
Comment (melamedbn) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force_-_Set_Verdict.yml:1]: Source Geolocation - Connection from an unusual country. - need to rephrase it to be more clear, something like:
Source Geolocation - Checks if the source geo-location has been seen in the past communicating with the brute-forced service.
* For all of them.

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml [hunk 9]
```diff
@@ -0,0 +1,3115 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "73":
+    id: "73"
+    taskid: 98bca17a-cf76-456d-8d3e-016d974cbbc4
+    type: condition
+    task:
+      id: 98bca17a-cf76-456d-8d3e-016d974cbbc4
+      version: -1
+      name: Is IP malicious?
```
Comment (melamedbn) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml:872]: Is **the** IP malicious?

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml [hunk 10]
```diff
@@ -0,0 +1,3115 @@
...
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+        simple: Do you want to perform containment using playbooks?
```
Comment (melamedbn) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml:1537]: Would you like to contain the IP and account using playbooks?

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml [hunk 11]
```diff
@@ -0,0 +1,3115 @@
+id: Cortex XDR - Possible External RDP Brute-Force
+version: -1
```
Comment (melamedbn) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml:2]: image
Didn't we want to remove the Get entity alerts for the user? we can use the User Investigation - Generic

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml [hunk 12]
```diff
@@ -0,0 +1,3115 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "93":
+    id: "93"
+    taskid: c6b2a205-bcd3-4140-8698-3601e1c729ea
+    type: playbook
+    task:
+      id: c6b2a205-bcd3-4140-8698-3601e1c729ea
+      version: -1
+      name: Cortex XDR - Get entity alerts by MITRE tactics
```
Comment (melamedbn) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml:1161]: If we keep it, change the name of the playbook.
Instead of '.... Get entity alerts...' use '...Get username related alerts...'

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml [hunk 13]
```diff
@@ -0,0 +1,3115 @@
...
+    taskid: 816caa2b-57e7-43ea-8a49-c2f2cc20cde9
+    type: playbook
+    task:
+      id: 816caa2b-57e7-43ea-8a49-c2f2cc20cde9
+      version: -1
+      name: Cortex XDR - Get entity alerts by MITRE tactics
+      description: |-
+        This playbook is part of the Cortex XDR by Palo Alto Networks’ pack. This playbook searches alerts related to specific entities from Cortex XDR, on a given timeframe, based on MITRE tactics.
+        Note: The playbook's inputs enable manipulating the execution flow. Read the input descriptions for details.
+      playbookName: Cortex XDR - Get entity alerts by MITRE tactics
```
Comment (melamedbn) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml:1238]: If we keep it, change the name of the playbook.
Instead of '.... Get entity alerts...' use '...Get host related alerts...'

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force_-_Set_Verdict.yml (hunk 8 above)
Comment (ArikDay) [reply Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force_-_Set_Verdict.yml:1]: to be accurate, it should detail the condition that the element will suspicious. I will rephrase it in this direction

**Review by melamedbn — APPROVED** [review]
